src/Generator/EpubGenerator.php

```diff
@@ -398,17 +403,23 @@ private function getXhtmlTitle( Book $book ) {
 	}
 
 	private function getXhtmlAbout( Book $book, $wsUrl ) {
-		$list = '<ul>';
-		$listBot = '<ul>';
-		foreach ( $book->credits as $name => $value ) {
-			if ( $value[ 'bot' ] !== null ) {
-				$listBot .= '<li>' . htmlspecialchars( $name, ENT_QUOTES ) . "</li>\n";
-			} else {
-				$list .= '<li>' . htmlspecialchars( $name, ENT_QUOTES ) . "</li>\n";
+		if ( !$book->options['credits'] ) {
+			$list = $this->intuition->msg( 'credits-default-message' ) ??
+					'credits-default-message missing';
```

Comment:
Why do we need to fall back to this?

Reply:
Phan will complain since the `msg` function might return null, and null cannot be passed to `Util::getXhtmlFromContent` as argument 2. I added a comment in the code so there is not confusion in the future.